src/components/routes/user-profile/Profile.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import makeStyles from '@material-ui/core/styles/makeStyles';
+import Card from '@material-ui/core/Card';
+import CardActionArea from '@material-ui/core/CardActionArea';
+import CardMedia from '@material-ui/core/CardMedia';
+import CardContent from '@material-ui/core/CardContent';
+import Typography from '@material-ui/core/Typography';
+import placeholder from '../../../assets/no_image_placeholder.png';
+
+const useStyles = makeStyles({
+  root: {
+    maxWidth: 400,
+    textAlign: 'left'
+  },
+  media: {
+    height: 200,
+    width: 220
+  }
+});
+
+export const Profile = ({ me }) => {
+  const classes = useStyles();
+
+  const { username, first_name, last_name, image_url } = me;
+
+  return (
+    <Card className={classes.root}>
+      <CardActionArea>
+        <CardMedia
+          className={classes.media}
+          image={image_url ? image_url : placeholder}
```

Comment:
let's use another placeholder for profile picture maybe, [this](https://www.google.com/search?q=profile+no+picture&tbm=isch&safe=strict&chips=q:profile+no+picture,g_1:facebook:FNjqG_fsEVY%3D&rlz=1C5CHFA_enHK859HK859&safe=strict&hl=en&ved=2ahUKEwjZnKDXlq3oAhWdxYsBHfjnAvkQ4lYoA3oECAEQGg&biw=1440&bih=821) might work 😂